packages/app-next/src/examples/graphiqlPlugin.tsx

```diff
@@ -46,3 +48,29 @@ export const graphiqlPlugin = createPlugin({
     },
   ],
 });
+
+export const GraphiqlPage = createPageExtension({
```

Comment:
I think this should replace the `GraphiqlPageExtension` above, and let's skip the custom `configSchema` and `inputs` here too imo. Could leave that for a separate example.